nixos/modules/services/admin/oddjobd.nix

```diff
@@ -0,0 +1,36 @@
+{ config, pkgs, lib, ... }:
+
+with lib;
+
+let
+  pkg = pkgs.oddjob;
+  cfg = config.services.oddjobd;
+in
+{
+  options.services.oddjobd = {
+    enable = mkOption {
+      type = types.bool;
+      default = false;
+      description = ''
+        Enable the oddjob service which allows scheduling jobs over DBUS.
+      '';
+    };
+  };
+
+  config = mkIf cfg.enable {
+    systemd.packages = [ pkgs.oddjob ];
+
+    systemd.services.oddjobd = {
+      wantedBy = ["multi-user.target"];
+      after = ["network.target"];
+      description = "DBUS Odd-job Daemon";
+      enable = true;
+      documentation = ["man:oddjobd(8)" "man:oddjobd.conf(5)"];
+      #serviceConfig = {
```

Comment:
Are you sure this needs to be commented out?

Reply:
No. But without it I think we try to create the service twice...

Reply:
Can you supress creation of the default .service file? Maybe nix is installing it when the package builds and then the option is installing it again with the mkIf

Reply:
Any reason we don't just use the one upstream provides?